lib/ansible/plugins/strategy/__init__.py

```diff
@@ -1015,9 +1024,9 @@ def _cond_not_supported_warn(self, task_name):
 
     def _execute_meta(self, task, play_context, iterator, target_host):
 
-        # meta tasks store their args in the _raw_params field of args,
-        # since they do not use k=v pairs, so get that
-        meta_action = task.args.get('_raw_params')
+        # old style meta tasks store their args in the _raw_params field of args,
+        # since they never use to use k=v pairs, now its recomended to use name k=v pair
```

Comment:
k=v pairs is never recommended but the comment here isn't (and wasn't) really about k=v so it's probably more clear to drop the k=v mention:
> old style meta tasks only had one value, the name of the meta action.  This would be stored, unparsed in _raw_params.  Meta tasks can now take separate fields for the action and options.  If so, the meta_action will be stored in the 'name' dictionary field.